admin/client/App/index.js

```diff
@@ -21,8 +21,6 @@ Keystone.User = listsByKey[Keystone.userList];
 const { adminPath } = Keystone;
 
 const doRender = () => {
-	const Routes = require('./Routes').default;
```

Comment:
What is the fix? I suppose react-router complains when the routes change during hot reload, but that doesn't mean we shouldn't do it…